infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -47,8 +47,8 @@ cd $SRC/chromium_tools
 git clone https://chromium.googlesource.com/chromium/src/tools/clang
 cd clang
 
-OUR_LLVM_REVISION=356356  # For manual bumping.
-FORCE_OUR_REVISION=1  # To allow for manual downgrades.
+OUR_LLVM_REVISION=342582  # For manual bumping.
```

Comment:
Maybe leave `356356` here, just in case we have to roll back again, that would make it easier.

Reply:
Done.